lib/class-wp-rest-styles-controller.php

```diff
@@ -39,11 +47,13 @@ public function get_url( $src, $ver, $handle ) {
 	 * @return array
 	 */
 	public function get_core_assets() {
-		$wp_styles = new WP_Styles();
-		wp_default_styles( $wp_styles );
-		$handles = wp_list_pluck( $wp_styles->registered, 'handle' );
-		$handles = array_values( $handles );
+		if ( null === $this->core_assets ) {
```

Comment:
Why cache has these?

Reply:
@spacedmonkey I noticed that this function is called twice during a single request and it's not harmful at all to cache it for the second execution. It's not particularly slow but basically no real need to run this code the second time.